manifest.toml

```
@@ -52,11 +63,14 @@ ram.runtime = "10M"
     [resources.ports]
     main.default = 1883
     main.exposed = "TCP"
+    websocket.default = 8883
+    ssl.default = 8884
+
 
     [resources.apt]
     packages = "mosquitto"
 
     [resources.apt.extras.mosquitto]
     repo = "deb https://repo.mosquitto.org/debian bullseye main"
```

```suggestion
    repo = "deb https://repo.mosquitto.org/debian __YNH_DEBIAN_VERSION__ main"
```

Available since YunoHost 11.2.11 (bump the requirement)